packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -1,7 +1,7 @@
 
 import { test, expect } from '@sand4rt/experimental-ct-web';
 import type { Page } from '@playwright/test';
-import { IconPlaceholder } from '@justeattakeaway/pie-icons-webc/IconPlaceholder';
+import { IconPlaceholder } from '@justeattakeaway/pie-icons-webc';
```

Comment:
This isn't ideal, but for some reason a direct import wasn't working in the tests. This at least aligns it with the 5 or so other component tests that import icons.

Reply:
What was the issue out of interest in the test?

Reply:
It's mostly complaining about Typescript stuff, like in the input browser tests it complains that PIEInputElement is being imported. I think it might be because we don't have a TypeScript step in the pie-webc-core build...